doc/en/how-to/fastapi-integration-testing-tutorial.rst

```diff
@@ -0,0 +1,219 @@
+FastAPI Integration Testing Tutorial
+==================================
+
+This tutorial demonstrates how to write integration tests for FastAPI applications using pytest.
+
+Prerequisites
+------------
+
+* Python 3.7+
+* pytest
+* FastAPI
+* SQLAlchemy
+* pytest-asyncio
```

Comment:
FastAPI already comes with a pytest plug-in, just use `@pytest.mark.anyio` you don't need to install an extra one